infrastructure/src/main/java/org/corfudb/infrastructure/log/InMemoryStreamLog.java

```diff
@@ -49,6 +52,7 @@ public synchronized void append(List<LogData> entries) {
 
             logCache.put(entry.getGlobalAddress(), entry);
             logMetadata.update(entry, false);
+            logSizeQuota.consume(entry.getSizeEstimate ());
```

Comment:
You can use the style template and get ride of these spaces between the method name and the empty round brackets